external/RSL_LITE/c_code.c

```diff
@@ -484,12 +484,12 @@ RSL_LITE_PACK ( int * Fcomm0, char * buf , int * shw0 ,
 	    MPI_Abort(MPI_COMM_WORLD, 99) ;
           }
           if ( typesize == 8 ) {
-            F_PACK_LINT ( buf, p+yp_curs, imemord, &js, &je, &ks, &ke, &is, &ie, 
+            F_PACK_LINT ( (long *)buf, (long *)(p+yp_curs), imemord, &js, &je, &ks, &ke, &is, &ie, 
                                                 &jms,&jme,&kms,&kme,&ims,&ime, &wcount ) ;
             yp_curs += wcount*typesize ;
           }
 	  else if ( typesize == 4 ) {
-            F_PACK_INT ( buf, p+yp_curs, imemord, &js, &je, &ks, &ke, &is, &ie,
+            F_PACK_INT ( (int * )buf, (int * )(p+yp_curs), imemord, &js, &je, &ks, &ke, &is, &ie,
```

Comment:
Is there a reason for using an extra space after the `*` here, but not for the `(long *)` casts?

Reply:
I believe it was so that 
`(long *)` and
`(int * )` 
evaluate to the same length of characters, but it could also just that I added it in the find and replace commands.